ha/core/controllers/pcs/cluster_controller.py

```diff
@@ -97,6 +97,22 @@ def _get_node_group(self) -> list:
                         for i in range(0, len(node_list), const.PCS_NODE_GROUP_SIZE)]
         return node_group
 
+    @controller_error_handler
+    def cluster_exists(self) -> dict:
+        """
+        Check if cluster exists.
+
+        Returns:
+            dict: Return dictionary. {"status": "", "msg":""}
+                status: Succeeded, Failed, InProgress
```

Comment:
I think we return only "Succeeded" here.

Reply:
Removed other